docs/3.0.0-beta.x/guides/schedule-article.md

```diff
@@ -0,0 +1,80 @@
+# Schedule article
+
+This guide will explain how to create an article schedule system.
+
+## Introduction
+
+This guide will use this [draft system](.draft.md) guide. So first read it to understand the draft/published status and the Article API that we will use in this guide.
+
+What we want here is to be able to set a publication date for an article, and at this date, switch the draft status to published.
+
+## Example
+
+For this example, we will have to add a `published_at` attribute to the Article Content Type.
+
+- Click on the Content Type Builder link in the left menu
+- Select the Article Content Type
+- Add another field
+  - `date` attribute named `published_at` with `datetime` type
+
+And add some data with different dates and status to be able to see a plublication happen.
+Some entries need to have `draft` status and a `published_at` date lower that your current date.
+
+The goal will be to verify every minute if there is `draft` articles that have a `published_at` lower that the current date.
+
+## Create a CRON task
+
+To execute a function every minutes, we will use CRON tasks.
+
+Here is the [full documentation](../concepts/configurations.md#cron-tasks) of this feature.
+
+**Path —** `./config/functions/cron.js`
+
+```js
+module.exports = {
+  '*/1 * * * *': () => {
+    console.log('1 minute later');
+  },
+};
+```
+
+<<<<<<< HEAD
+Make sure the enabled cron config is set to true in ./config/environments/**/server.json file.
```

Comment:
single line code blocks around the file path

Reply:
(also it looks like there is some github merge HEAD lines here that need to be removed)